pyaqueduct/client/client.py

```diff
@@ -309,6 +313,26 @@ def remove_tag_from_experiment(self, experiment_uuid: UUID, tag: str) -> Experim
         logging.info("Removed tag %s from experiment <%s>", tag, experiment_obj.title)
         return experiment_obj
 
+    def remove_files_from_experiment(self, experiment_uuid: UUID, files: List[str]) -> None:
+        """
+        Remove files from an experiment
+
+        Args:
+            experiment_uuid: UUID of experiment for which files has to be removed from.
+            files: List of file names to be removed.
+
+        """
+
+        remove_url = f"{self.url}/files/{str(experiment_uuid)}/delete_files"
```

Comment:
```suggestion
        remove_url = f"{self.url}/files/{experiment_uuid}/delete_files"
```

Maybe this is also ok?

Reply:
Yes @samiralavi , as we are already using f-string, we don't need to use `str` for conversion to string